fst-bin/Cargo.toml

```diff
@@ -21,15 +22,13 @@ path = "src/main.rs"
 [dependencies]
 bit-set = "0.4"
 chan = "0.1"
-csv = "1.0.0-beta.3"
-docopt = "0.8"
-fst = { path = "..", version = "0.3" }
-fst-regex = { path = "../fst-regex", version = "0.2" }
-fst-levenshtein = { path = "../fst-levenshtein", version = "0.2" }
+csv = "1.1.3"
+docopt = "1.1"
+fst = { path = "..", version = "0.3", features = ["levenshtein"] }
 lines = "0.0"
```

Comment:
I believe having a version number of 0.0 as a dependency can cause issues with semver. Perhaps this could be updated to the specific version number as well?

Reply:
It's been this way for a long time and it has been fine. This is using [`lines`](https://crates.io/crates/lines) from crates.io, whose latest release is `0.0.6`. So `0.0` is fine here I think?

In any case, I'll be refactoring `fst-bin`, and it will no longer use `lines`. Instead, it will use [`bstr`](https://docs.rs/bstr).

Reply:
I think that is largely due to lines not being updated much. Going from say 0.0.6 to 0.0.7 can be a breaking change so it is possible to run into issues in the future. If you are getting rid of the dependency anyways it doesn't really matter then.